src/Dapr.Actors/Client/ActorProxyFactory.cs

```diff
@@ -12,52 +12,59 @@ namespace Dapr.Actors.Client
     /// <summary>
     /// Represents a factory class to create a proxy to the remote actor objects.
     /// </summary>
-    internal class ActorProxyFactory : IActorProxyFactory
+    public class ActorProxyFactory : IActorProxyFactory
     {
         private readonly IDaprInteractor daprInteractor;
+        private ActorProxyOptions defaultOptions = new ActorProxyOptions();
+
+        /// <inheritdoc/>
+        public ActorProxyOptions DefaultOptions
+        {
+            get => this.defaultOptions;
+            set => this.defaultOptions = value ??
+                    throw new ArgumentNullException(nameof(DefaultOptions), $"{nameof(ActorProxyFactory)}.{nameof(DefaultOptions)} cannot be null");
+        }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ActorProxyFactory"/> class.
-        /// TODO: Accept Retry settings.
         /// </summary>
         public ActorProxyFactory()
         {
-            // TODO: Allow configuration of serialization and client settings.
             this.daprInteractor = new DaprHttpInteractor();
         }
 
-        /// <inheritdoc/>
-        public TActorInterface CreateActorProxy<TActorInterface>(ActorId actorId, string actorType)
-            where TActorInterface : IActor
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ActorProxyFactory"/> class.
+        /// </summary>
+        public ActorProxyFactory(ActorProxyOptions options) : this()
```

Comment:
Note that this constructor isn't needed anymore since the property is get/set. I'd kinda prefer a single paradigm - either a constructor and a get-only property or a no-arg constructor and a get/set property.